### PR TITLE
Update installation instructions to use cuDNN v5 across builds.

### PR DESCRIPTION
**Pull request by yifeif**

### CODE REVIEW

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -581,15 +580,14 @@ Install the toolkit into e.g. `/usr/local/cuda`
...
-installing TensorFlow from sources). If setting up on Mac, download cuDNN v5.
+Download cuDNN v5.
 
 Uncompress and copy the cuDNN files into the toolkit directory.  Assuming the
 toolkit is installed in `/usr/local/cuda`, run the following commands (edited
 to reflect the cuDNN version you downloaded):
 
 ``` bash
-tar xvzf cudnn-7.5-linux-x64-v4.tgz
+tar xvzf cudnn-7.5-linux-x64-v5.0-ga.tgz
```
Comment (vrv) [inline tensorflow/g3doc/get_started/os_setup.md]: we should probably recommend v5.1 in the example here...

File: tensorflow/g3doc/get_started/os_setup.md [hunk 2]
```diff
@@ -732,8 +730,7 @@ Configuration finished
 
 This creates a canonical set of symbolic links to the Cuda libraries on your system.
 Every time you change the Cuda library paths you need to run this step again before
-you invoke the bazel build command. For the cuDNN libraries, use '6.5' for R2, '7.0'
-for R3, and '4.0.4' for R4-RC.
+you invoke the bazel build command. For the cuDNN libraries, use '7.0' for R3, and '4.0.4' for R4-RC.
```
Comment (vrv) [inline tensorflow/g3doc/get_started/os_setup.md]: This should probably be updated to be 4.0.7 for R4, since it's out of RC and was bumped up to 4.0.7.

**Comment by vrv** [comment]
other than two small suggestions, LGTM

**Comment by yifeif** [comment]
Thanks @vrv! I have made the changes accordingly.

**Pull request merged**